ngraph_onnx/onnx_importer/ops_bridge/opset_01.py

```diff
@@ -462,34 +462,27 @@ def Greater(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> N
 def And(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
     """Perform the `and` logical operation elementwise on two input tensors."""
     left, right = broadcast_for_binary_operation(onnx_node, ng_inputs)
-    left = ng.convert(ng.not_equal(left, 0), int)
-    right = ng.convert(ng.not_equal(right, 0), int)
-    return ng.convert(left * right, bool)
+    return ng.logical_and(ng.convert(left, bool), ng.convert(right, bool))
```

Comment:
ONNX documentation specify input type as bool.
Why do we have to convert inputs to bool?

Reply:
That's right onnx documentation says it is bool. However in our tests we passed integers which caused errors :) I've not checked doc so it's my mistake.